diff -up kde-settings-4.3.1/usr/share/kde-settings/kde-profile/default/share/config/konquerorrc.orig kde-settings-4.3.1/usr/share/kde-settings/kde-profile/default/share/config/konquerorrc
--- kde-settings-4.3.1/usr/share/kde-settings/kde-profile/default/share/config/konquerorrc.orig	2013-06-17 18:08:46.922854902 +0200
+++ kde-settings-4.3.1/usr/share/kde-settings/kde-profile/default/share/config/konquerorrc	2013-06-17 18:09:08.384802703 +0200
@@ -2,6 +2,7 @@
 EnableJavaScript=true
 EnableJSDebugOutput=false
 EnablePlugins=true
+EnableJava=false
 
 [HTML Settings]
 AutoDelayedActions=true
